server/routes/api/admin.ts

```diff
@@ -0,0 +1,27 @@
+import express, { NextFunction, Request, Response } from 'express';
+
+import { auth } from '../../middleware/auth';
+
+import * as usersController from '../../controllers/users/users';
+
+const router = express.Router();
+
+// @route   GET api/admin/userList
+// @desc    Return user list
+// @access  Private
+router.get(
+    '/userList',
+    (req: Request, res: Response, next: NextFunction) => auth(req, res, next, true, true),
```

Comment:
what does it mean true, true?

Reply:
@mras2303 
4th argument means that route including authencitation for 'emailVerified' field,
5th argument including authencitation for admin

Reply:
is there any possibility to write it in better way?

Reply:
You tell me :D I gonna try find out

Reply:
I read that if we're using more than 1 optional parameter, we should use object so we can do smth like:
**in auth.ts:**
`
export const auth = (
	req: Request,
	res: Response,
	next: NextFunction,
	{authEmail = true, authAdmin = false} : {authEmail?: boolean, authAdmin?: boolean },
) => {....
`
**and then in route without parameters:**
`(req: Request, res: Response, next: NextFunction) => auth(req, res, next, {}),`
**with 1 any optional parameter:**
`(req: Request, res: Response, next: NextFunction) => auth(req, res, next, {authAdmin: true}),`

Reply:
To further discuss

Reply:
I think object with settings params is great option

Reply:
i've changed this and now it's pretty short and clean